content/guides/additional/client.md

```diff
@@ -0,0 +1,2154 @@
++++
+title = "Clients"
+weight = 11
++++
+
+## Introduction
+
+In this tutorial, we will take an off-the-shelf JavaScript game which runs in the browser and connect it to an Urbit back-end.  This page assumes that you have completed some version of Hoon School and App School, whether the [live courses](/courses) or the [written docs](/guides/core/hoon-school/A-intro).
+
+_Flappy Bird_ is an "insanely irritating, difficult and frustrating game which combines a super-steep difficulty curve with bad, boring graphics and jerky movement" ([Huffington Post](https://web.archive.org/web/20140205084251/http://www.huffingtonpost.com/2014/02/03/flappy-bird-tips_n_4717406.html)).  We are going to implement `%flap`, a _Flappy Bird_ leaderboard using `%pals` and an off-the-shelf [FOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software) JavaScript game.  The approach given in this tutorial will apply to any game which is primarily run in the browser and has some persistent state to retain across sessions or communicate between players at discrete intervals.  Direct player-v.-player games will require other techniques to implement.
```

Comment:
this is not FOSS, it has no license. The original flappy bird is not FOSS, the original developer just hasn't taken action against people who clone his app unless they're making money, so it's a grey area. I would remove any mention of it being FOSS and also change the license in the docket file to an empty string or something

Reply:
to be clear I think it will be fine to use flappy bird like this, the developer has previously made encouraging statements when people have used his code for non-profit purposes, but I would just be careful about license assertions